modules/payment/src/Plugin/Commerce/PaymentGateway/OffsitePaymentNotificationInterface.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Drupal\commerce_payment\Plugin\Commerce\PaymentGateway;
+
+use Symfony\Component\HttpFoundation\Request;
+
+/**
+ * Defines the interface for gateways which receive payment notifications.
+ */
+interface OffsitePaymentNotificationInterface {
```

Comment:
Let's rename this to SupportsNotificationsInterface, that would fit with the others (refunds, authorizations, etc)

Reply:
Sounds good. Done =)